postgres_mixin/alerts/postgres.libsonnet

```diff
@@ -278,7 +283,18 @@
               description: '{{ $labels.instance }} has too many dead tuples, which may lead to inefficient query performance. Consider vacuuming the database.',
               summary: 'PostgreSQL has too many dead tuples.',
             },
-            expr: '(sum without(relname) (pg_stat_user_tables_n_dead_tup{asserts_env!="", db_name!~"template.*|^$"}) > 10000) / ((sum without(relname) (pg_stat_user_tables_n_live_tup{asserts_env!="", db_name!~"template.*|^$"}) + sum without(relname)(pg_stat_user_tables_n_dead_tup{asserts_env!="", db_name!~"template.*|^$"})) > 0) >= 0.1 unless on(instance, asserts_env, asserts_site) (pg_replication_is_replica{asserts_env!=""} == 1)',
+            expr: |||
+              (sum without(relname) (
+                pg_stat_user_tables_n_dead_tup{%(dbNameFilter)s}
+              ) > 10000) /
+              ((sum without(relname) (
+                pg_stat_user_tables_n_live_tup{%(dbNameFilter)s}
+              ) + sum without(relname)(
+                pg_stat_user_tables_n_dead_tup{%(dbNameFilter)s}
+              )) > 0) >= 0.1 unless on(instance) (
+                pg_replication_is_replica{} == 1
+              )
+            ||| % $._config,
```

Comment:
Description for this query definitely needs a bit more meat to explain what's going on here

Reply:
Also removing this alert `PostgresHasTooManyDeadTuples`

since `pg_stat_user_*` metrics do not exist in the exporter
https://github.com/grafana/postgres_exporter/blob/f59d4af692561afd1dc3d9c749924c8ea7436ae3/cmd/postgres_exporter/postgres_exporter.go#L249